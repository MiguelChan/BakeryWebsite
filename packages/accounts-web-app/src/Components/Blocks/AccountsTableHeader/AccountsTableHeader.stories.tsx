import React from 'react';
import {
  Story,
  Meta,
} from '@storybook/react';
import { AccountsTableHeader } from './AccountsTableHeader';
import { Table } from '@mui/material';

export default {
  title: 'Components/Blocks/AccountsTableHeader',
  component: AccountsTableHeader,
  decorators: [
    (Story) => (
      <Table>
        <thead>
          <Story />
        </thead>
      </Table>
    ),
  ],
} as Meta;

const Template: Story = () => <AccountsTableHeader />;

export const Primary = Template.bind({});
