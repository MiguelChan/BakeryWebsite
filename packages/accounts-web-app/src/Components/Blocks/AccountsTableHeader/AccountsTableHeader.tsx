import {
  TableCell,
  TableRow,
} from '@mui/material';
import React from 'react';

/**
 * Defines the AccounsTable Header.
 * @returns
 */
export const AccountsTableHeader: React.FunctionComponent = () => (
  <TableRow>
    <TableCell>
      Id
    </TableCell>
    <TableCell>
      Nombre de cuenta
    </TableCell>
    <TableCell>
      Tipo de Cuenta
    </TableCell>
  </TableRow>
);
