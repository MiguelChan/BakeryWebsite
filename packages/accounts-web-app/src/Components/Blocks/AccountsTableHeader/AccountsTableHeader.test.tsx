import React from 'react';
import {
  render,
  RenderResult,
} from '@testing-library/react';
import { Table, TableHead } from '@mui/material';
import { AccountsTableHeader } from './AccountsTableHeader';

describe('AccountsTableHeader', () => {
  const setupComponent = (): RenderResult => {
    const Component = (
      <Table>
        <TableHead>
          <AccountsTableHeader />
        </TableHead>
      </Table>
    );
    return render(Component);
  };

  it('Should display the data', () => {
    const component = setupComponent();
    expect(component.container).toMatchSnapshot();
  });
});