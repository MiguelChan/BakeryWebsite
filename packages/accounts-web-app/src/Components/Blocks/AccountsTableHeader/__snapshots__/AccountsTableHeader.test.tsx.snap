// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountsTableHeader Should display the data 1`] = `
<div>
  <table
    class="MuiTable-root css-rqglhn-MuiTable-root"
  >
    <thead
      class="MuiTableHead-root css-15wwp11-MuiTableHead-root"
    >
      <tr
        class="MuiTableRow-root MuiTableRow-head css-1q1u3t4-MuiTableRow-root"
      >
        <th
          class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
          scope="col"
        >
          Id
        </th>
        <th
          class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
          scope="col"
        >
          Nombre de cuenta
        </th>
        <th
          class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
          scope="col"
        >
          Tipo de Cuenta
        </th>
      </tr>
    </thead>
  </table>
</div>
`;
